src/components/Permalink/index.tsx

```diff
@@ -28,6 +40,8 @@ export interface PermalinkProps extends Partial<DefaultPermalinkProps> { }
 
 export const Permalink: React.FC<PermalinkProps> = () => {
 
+  const [permalink, setPermalink] = useState('');
```

Comment:
This will result in an empty link on initial render. We might want to call `PermalinkUtil.getLink` instead of providing an empty string.